### PR TITLE
Create pythonapp.yml

### PR DESCRIPTION
**Pull request by ajcsathy4**

### CODE REVIEW

**Review by TDunn8 — APPROVED** [review]
Looks good!

**Review by TyrannicBee — APPROVED** [review]
Looks good

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```
Comment (steven-hh-ding) [inline .github/workflows/pythonapp.yml:19]: @TyrannicBee 

it cannot find `requirements.txt` file. it is not under the root of your git repo. you may add `cd CI-Python` before `pip install`

**Review by steven-hh-ding — COMMENTED** [review]
requirement.txt not found causing action failure.

File: .github/workflows/pythonapp.yml (hunk 1 above)
Comment (steven-hh-ding) [reply .github/workflows/pythonapp.yml:19]: also @TDunn8

Comment (steven-hh-ding) [reply .github/workflows/pythonapp.yml:19]: if it is the module missing (e.g. qa327), you may want to `cd` as well, before `pytest`

**Pull request merged**